### PR TITLE
Aden 2443 flite tag

### PR DESCRIPTION
**Pull request by PiotrGackowski**
Tests for replacement verbatim tag in features connected to Flite quizes.

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 30 issues:
* MINOR 30 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Define a constant instead of duplicating this literal "naruto" 3 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L45) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "src/test/resources/adsResources/wikia_skin_right.png" 9 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L213) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "adtest-pad" 3 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L520) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "%D0%9A%D0%B2%D0%B5%D1%81%D1%82%D1%8B_%28Skyrim%29" 3 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L98) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "AAAAAA" 9 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L214) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "Wikia" 4 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L79) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "ru.elderscrolls" 3 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L98) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "SyntheticTests/Interstitial/NotScalable" 4 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L538) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "zh.pad" 4 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L53) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "wikia" 7 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L79) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "SyntheticTests/ProvidersChain" 4 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L420) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "DirectGpt; RemnantGpt; Liftium" 3 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L421) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "CORP_TOP_LEADERBOARD" 4 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L79) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "gameshub" 3 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L83) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "adtest" 45 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L33) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "HUB_TOP_LEADERBOARD" 8 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L83) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "elderscrolls" 3 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L67) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "FFFFFF" 8 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L215) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "TOP_LEADERBOARD" 10 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L165) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "126608052" 4 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L165) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "SyntheticTests/Interstitial" 4 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L531) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "monsterhunter" 5 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L102) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "MH3U:_Monsters" 3 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L102) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "wka.hub/_life_hub//hub" 4 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L85) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "leagueoflegends" 3 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L46) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "yugioh" 3 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L44) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "src/test/resources/adsResources/wikia_skin_left.png" 9 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L212) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "TOP_RIGHT_BOXAD" 3 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L177) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "SyntheticTests/Krux/Page_1" 4 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L458) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "adtest-pluto" 8 times.](https://github.com/Wikia/selenium-tests/blob/6c95876e999afee2792b25d7db9b1c27be5bfed6/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L104) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFliteTagOasis.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    AdsBaseObject wikiPage = new AdsBaseObject(driver, testedPage);
+    wikiPage.checkFliteTag();
+  }
+
+  @Test(
+      dataProviderClass = AdsDataProvider.class,
+      dataProvider = "fliteTagBroken",
+      groups = {"AdsFliteTagOasis"}
+  )
+  public void adsFliteTagBrokenWidthOasis(String wikiName, String article, String error) {
```
Comment (drets) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFliteTagOasis.java]: `adsFliteTagBrokenWidthOasis` -> `adsFliteTagBrokenOasis` ?

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFliteTagOasis.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+/**
+ * @ownership: AdEngineering
+ */
+public class TestAdsFliteTagOasis extends TemplateNoFirstLoad {
+
+
+  @Test(
+      dataProviderClass = AdsDataProvider.class,
+      dataProvider = "fliteTag",
+      groups = {"AdsFliteTagOasis"}
```
Comment (drets) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFliteTagOasis.java]: I would remove `{` and `}`, but it's up to you.

**Comment by drets** [comment]
Except minor comments, LGTM :+1:

**Pull request merged**